artistools/inputmodel/describeinputmodel.py

```diff
@@ -149,35 +149,40 @@ def main(args: argparse.Namespace | None = None, argsraw: t.Sequence[str] | None
     mass_actinides_isosum = 0.0
     speciesmasses: dict[str, float] = {}
     for column in dfmodel.columns:
-        if column.startswith("X_"):
-            species = column.replace("X_", "")
-            speciesabund_g = np.dot(dfmodel[column], dfmodel["cellmass_grams"])
+        if not column.startswith("X_"):
+            continue
 
-            species_mass_msun = speciesabund_g / 1.989e33
+        if column.startswith("X_n"):  # don't confuse neutrons with Nitrogen
+            continue
 
-            atomic_number = at.get_atomic_number(species.rstrip("0123456789"))
+        species = column.replace("X_", "")
+        speciesabund_g = np.dot(dfmodel[column], dfmodel["mass_g"])
 
-            if species[-1].isdigit():
-                # isotopic species
+        species_mass_msun = speciesabund_g / 1.989e33
```

Comment:
Would it not be better to define Msun rather than write it out everywhere? Same with CLIGHT